src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/signup/UserProfilePageObject.java

```diff
@@ -107,9 +106,23 @@ public void clickRemoveAvatar() {
     PageObjectLogging.log("clickRemoveAvatar", "avatar remove button clicked", true);
   }
 
-  public void verifyAvatar(String fileName) {
-    wait.forElementVisible(By.cssSelector(avatarSelector.replace("%imageName%", fileName)));
-	  PageObjectLogging.log("verifyAvatar", "Desired avatar is visible on user profile page", true);
+  public void verifyAvatar() {
+    wait.forElementVisible(avatar);
+    PageObjectLogging.log("verifyAvatar", "Desired avatar is visible on user profile page", true);
+  }
+
+  public void verifyAvatarChanged(String oldUrl) {
```

Comment:
Yeah the name might be confusing, the function actually waits for the "oldUrl" value to stop being set. I would rename it to urlToBeChanged or something like that - or add a comment to the function to avoid confusion